tests/sqf/while.sqf

```diff
@@ -0,0 +1,10 @@
+[   ["assertEqual",     { private _i = 0; while{ false } do { _i = 1 }; _i }, 0],				// while CODE do CODE: condition false
+	["assertIsNil", 	{ while{ false } do {} } ],												// while CODE do CODE: condition false
+	["assertException", { while{ } do {} } ],													// while CODE do CODE: condition empty
+	["assertException", { while{ "" } do {} } ],												// while CODE do CODE: condition not BOOL
+	["assertIsNil", 	{ while{ nil } do {} } ],												// while CODE do CODE: condition nil
```

Comment:
Use 4 space indent. Just like in runTests.sqf
You are really mixing tabs and spaces in here alot. Also for the indent after the comma